lib/stages/testDataGen.cpp

```diff
@@ -68,6 +69,10 @@ testDataGen::testDataGen(Config& config, const std::string& unique_name,
     using namespace std::placeholders;
     restServer::instance().register_post_callback(
         endpoint, std::bind(&testDataGen::rest_callback, this, _1, _2));
+
+    // Create stat tracker
+    kotekan::KotekanTrackers& KT = kotekan::KotekanTrackers::instance();
```

Comment:
I think you should already be using the `kotekan` namespace, so the first part of this is likely not needed.

Reply:
There will be error of unknown type name, so I just leave it like this.

Reply:
Yeah you are right, this is needed.